tests/integration/module-resolution/module-resolution.test.ts

```diff
@@ -89,6 +69,22 @@ describe('module resolution', async () => {
 		});
 	});
 
+	test('node built-ins (both from userland and external dependencies)', async () => {
```

Comment:
So really node built-ins should fail unless we have nodejs_compat turned on.
Not sure, therefore, how I feel about this test...

Reply:
Oh but I see you did turn it on in the wrangler.toml...
Which means that the non-polyfilled modules are available.